apisix/plugins/kafka-logger.lua

```diff
@@ -99,6 +99,7 @@ local schema = {
         },
         key = {type = "string"},
         timeout = {type = "integer", minimum = 1, default = 3},
+        max_body_len = {type = "integer", minimum = 1048, default = 1048000},
```

Comment:
```suggestion
        message_max_bytes = {type = "integer", minimum = 1048, default = 1048588},
```

Reply:
keep the same as https://kafka.apache.org/documentation/#brokerconfigs_message.max.bytes